srai/plotting/plotly_wrapper.py

```diff
@@ -19,6 +20,69 @@
 import_optional_dependencies(dependency_group="plotting", modules=["plotly"])
 
 
+def plot_numeric_data(
+    regions_gdf: gpd.GeoDataFrame,
+    data_column: str,
+    data_df: Optional[Union[gpd.GeoDataFrame, pd.DataFrame]] = None,
+    colormap: Union[str, List[str]] = px.colors.sequential.Sunsetdark,
+    return_plot: bool = False,
+    mapbox_style: str = "open-street-map",
+    mapbox_accesstoken: Optional[str] = None,
+    renderer: Optional[str] = None,
+    zoom: Optional[float] = None,
+    height: Optional[float] = None,
+    width: Optional[float] = None,
+) -> Optional[go.Figure]:
+    """
+    Plot numeric data on a map using Plotly library.
+
+    For more info about parameters, check https://plotly.com/python/mapbox-layers/.
+
+    Args:
+        regions_gdf (gpd.GeoDataFrame): Region indexes and geometries to plot.
+        data_column (str): Column name with numeric data to plot.
+        data_df (Optional[Union[gpd.GeoDataFrame, pd.DataFrame]], optional): Data to plot.
+            If `None`, only regions will be plotted. Defaults to None.
+        colormap (Union[str, List[str]], optional): Colormap to use for plotting.
+            Can be a string with a name of a Plotly colormap, or a list of colours.
+            Defaults to px.colors.sequential.Sunsetdark.
+        return_plot (bool, optional): Flag whether to return the Figure object or not.
+            If `True`, the plot won't be displayed automatically. Defaults to False.
+        mapbox_style (str, optional): Map style background. Defaults to "open-street-map".
+        mapbox_accesstoken (str, optional): Access token required for mapbox based map backgrounds.
+            Defaults to None.
+        renderer (str, optional): Name of renderer used for displaying the figure.
+            For all descriptions, look here: https://plotly.com/python/renderers/.
+            Defaults to None.
+        zoom (float, optional): Map zoom. If not filled, will be approximated based on
+            the bounding box of regions. Defaults to None.
+        height (float, optional): Height of the plot. Defaults to None.
+        width (float, optional): Width of the plot. Defaults to None.
+    """
+    regions_gdf_copy = regions_gdf.copy()
+    regions_gdf_copy[REGIONS_INDEX] = regions_gdf_copy.index
```

Comment:
When I pass a geodataframe with regions from our regionalizer I get an error, because we set index name to `REGIONS_INDEX`. It results in an error on merge, because there is a column and index with the same name in `regions_gdf_copy`

```
ValueError: 'region_id' is both an index level and a column label, which is ambiguous.
```